packages/common/src/hooks/useCoinflowAdapter.ts

```diff
@@ -29,12 +30,33 @@ export const useCoinflowAdapter = () => {
           publicKey: wallet.publicKey,
           sendTransaction: async (transaction: Transaction) => {
             transaction.partialSign(wallet)
-            const res = await connection.sendRawTransaction(
-              transaction.serialize(),
-              // Skip preflight as this causes indeterminism between coinflow's
-              // RPC and our provided RPC.
-              { skipPreflight: true }
-            )
+            const transactionHandler = new TransactionHandler({
+              connection,
+              useRelay: false
+            })
+            const { res, error, errorCode } =
+              await transactionHandler.handleTransaction({
+                instructions: transaction.instructions,
+                recentBlockhash: transaction.recentBlockhash,
+                skipPreflight: true,
+                feePayerOverride: transaction.feePayer,
+                signatures: transaction.signatures.map((s) => ({
+                  signature: s.signature!, // already completely signed
+                  publicKey: s.publicKey.toBase58()
+                }))
+              })
+            if (!res) {
```

Comment:
Any use in debug-logging the transactions for the time being in the event that it fails to confirm but succeeded here?

Reply:
wdym? the retry logic confirms the tx

Reply:
That answers my question, sounds like there isn't any use!
We can always grab it from chain by signature and inspect it there.